internal/api/keppel/accounts_test.go

```diff
@@ -1633,6 +1656,13 @@ func TestDeleteAccount(t *testing.T) {
 		}
 	}
 
+	imageList := test.GenerateImageList(image.Manifest)
```

Comment:
I changed the type signature of the GenerateImageList function [yesterday](https://github.com/sapcc/keppel/commit/6dd6c27a03b377b8bdafbe6ccb8b625ff462807c), so this needs to be adapted:
```suggestion
	imageList := test.GenerateImageList(image)
```